service.js

```diff
@@ -39,48 +61,51 @@ module.exports = (models, modelName, req, res) => {
     return {count: await model.count(record)}
   }
 
-  return {
```

Comment:
Everything from here up until the end of the PR was just me changing the order of things, but no internal changes are present.